source/_integrations/habitica.markdown

```diff
@@ -20,40 +20,42 @@ ha_config_flow: true
 ha_integration_type: integration
 ---
 
-This integration allows you to monitor and manage your Habitica profile. This integration exposes the [Habitica's API](https://habitica.com/apidoc/) as a Home Assistant service. It supports multiple users and allows you to automate checking out your habits and daily tasks or casting magics using Home Assistant.
+The Habitca {% term integration %} enables you to monitor your adventurer's progress and stats in Home Assistant and seamlessly integrates your to-do's and daily tasks.  
```

Comment:
Are there any Prerequisites? Do I need an account, user name, API key etc. before configuring the integration in Home Assistant?

Reply:
There haven't been any changes to the setup process but it is described in the config flow how to authenticate with API key and token. Currently waiting for a PR to be approved that will further streamline the setup process of the integration:
- https://github.com/home-assistant/core/pull/117622

I'd suggest to add a section about setup and prerequisites to the documentation after that one was merged.

Reply:
I agree, it is outside the scope of this PR. Maybe together with 117622?